wagtail/locks.py

```diff
@@ -38,8 +38,7 @@ def __init__(self, page):
     def for_user(self, user):
         if getattr(settings, "WAGTAILADMIN_GLOBAL_PAGE_EDIT_LOCK", False):
             return True
-        else:
-            return user.pk != self.page.locked_by_id
+        return user.pk != self.page.locked_by_id
```

Comment:
This does not fit our preferred coding style - see https://github.com/wagtail/wagtail/pull/8734#discussion_r904991844.